assignments/A2/Nim_Game.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+
+public class Nim_Game{
+	
+	
+	public static void main(String[] args){
+		nim();
+	}
+	
```

Comment:
Please, add block comments before all your methods, makes it easier for you to remember what you were doing when you revisit this code anytime later. Plus, it saves me review time too